packages/mc-scripts/src/utils/get-config-diff.spec.ts

```diff
@@ -17,342 +21,538 @@ const createTestConfig = (customConfig = {}) => ({
+        expect(getCustomApplicationConfigDiff(oldConfig, newConfig))
+          .toMatchInlineSnapshot(`
+          "submenuLink changed
+            menu link "my-test-app/new" changed
+              defaultLabel changed: <color-red>New Avenger</color-red> => <color-green>Add Avenger</color-green>
+              permissions changed
+                applied permission removed: <color-red>ManageMyTestApp</color-red>
+              labelAllLocales changed
+                locale "de" changed: <color-red>Neu Avenger</color-red> => <color-green>Avenger hinzufügen</color-green>
+                locale added: <color-green>it</color-green>
+                locale removed: <color-red>en</color-red>"
+        `);
+      });
     });
   });
-
-  describe('for "submenuLinks"', () => {
-    it('should display diff for "submenuLinks" when adding a new menu link', () => {
-      const oldConfig = createTestConfig({
-        submenuLinks: [],
+});
+describe('Custom View Config Diff', () => {
+  describe('when there are no changes', () => {
+    it('should display no diff', () => {
+      expect(
+        getCustomViewConfigDiff(
+          createTestCustomViewConfig(),
+          createTestCustomViewConfig()
+        )
+      ).toBe('');
+    });
+  });
+  describe('when there are changes', () => {
+    it('should display diff if "defaultLabel", "description", "url" changed', () => {
+      const newConfig = createTestCustomViewConfig({
+        defaultLabel: 'updated test name',
+        description: 'updated description',
+        url: 'https://updated-test.com',
+      });
+      expect(getCustomViewConfigDiff(createTestCustomViewConfig(), newConfig))
+        .toMatchInlineSnapshot(`
+        "defaultLabel changed: <color-red>test name</color-red> => <color-green>updated test name</color-green>
+        description changed: <color-red>test description</color-red> => <color-green>updated description</color-green>
+        url changed: <color-red>https://test.com</color-red> => <color-green>https://updated-test.com</color-green>"
+      `);
+    });
+    it('should display diff if "description" is removed', () => {
+      const newConfig = createTestCustomViewConfig({
+        description: undefined,
       });
+      expect(
+        getCustomViewConfigDiff(createTestCustomViewConfig(), newConfig)
+      ).toMatchInlineSnapshot(
+        `"description removed: <color-red>test description</color-red>"`
+      );
+    });
 
-      const newConfig = createTestConfig({
-        submenuLinks: [
+    it('should display diff for "permissions"', () => {
+      const oldConfig = createTestCustomViewConfig({
+        permissions: [
           {
-            uriPath: 'my-test-app/new',
-            defaultLabel: 'New Avenger',
-            permissions: ['ManageMyTestApp'],
-            labelAllLocales: [
-              {
-                locale: 'de',
-                value: 'Neu Avenger',
-              },
-            ],
+            name: 'viewMyTestView',
```

Comment:
Nit:

```suggestion
            name: 'view',
```

Same for the others